src/hooks/transactions/useGetLastTransactionHash.ts

```diff
@@ -0,0 +1,26 @@
+import { useGetPendingTransactions } from 'hooks/transactions/useGetPendingTransactions';
+
+export const useGetLastTransactionHash = () => {
+  const { hasPendingTransactions, pendingTransactions } =
+    useGetPendingTransactions();
```

Comment:
scenario:
1. Make a transaction
2. Sign transaction
3. getLatestTransactionHash --> OK
4. Transaction gets resolved
5. getLatestTransactionHash --> empty

I think getLatestTransactionHash should return a hash if one or more transactions have been processed